client/src/utils/convertDate.js

```diff
@@ -23,5 +23,5 @@ export const convertDate = (dateString) => {
     time = `${hours}:${minutes}AM`
   }
   const convertedDateTime = `${month} ${day}, ${year} at ${time}`
-  return <p className='my-0 small-font'>{convertedDateTime}</p>
+  return convertedDateTime
```

Comment:
can you just `return `${month} ${day}, ${year} at ${time}`` instead of assigning it a variable and then returning that variable, more strict linters wouldn't let you do it the way you are currently doing it.